### DISCUSSION TITLE
colours are not correct in case of SBT_NATIVE_CLIENT=true

### DISCUSSION THREAD

**Discussion by robstoll** [sbt/sbt#6859] [Q&A]
I am on Ubuntu 20.04 using urxvt
Unfortunately, the colours sbt is shows does not work correctly when I use SBT_NATIVE_CLIENT=true
e.g. `[error]` is no longer red. Everything works as expected when I run sbt with SBT_NATIVE_CLIENT=false.

Any hints how to solve this issue?

**Comment by eed3si9n** [comment]
I am not sure if we can get to it, but it sounds like a bug. Could you open this as an issue please?
